tasks/aiida-prepare.yml

```diff
@@ -75,9 +75,9 @@
   pip:
     name:
     - pip~=20.0
-    - setuptools~=36.2
+    - setuptools
     - wheel
-    - numpy==1.17.4
+    - cython  # required by pymatgen
```

Comment:
So, to recap: I think the problem is that pymatgen still specifies its *build* dependencies (numpy) via an antiquated method, which results in `easy_install` being used instead of pip, which then requires numpy to be built (rather than just using a wheel).
I.e. just putting numpy here should also solve the problem and would be faster.

Reply:
Ok I'll give it a go thanks:
pymatgen does have numpy as a setup dependency not cython: https://github.com/materialsproject/pymatgen/blob/cea7587e40cff1d8b476601176b034d753301971/setup.py#L108

although feels weird that this is the dependency to build the cython files: https://github.com/materialsproject/pymatgen/blob/cea7587e40cff1d8b476601176b034d753301971/setup.py#L164

Reply:
works 👍